src/api/libcellml/variable.h

```diff
@@ -208,6 +208,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
      */
     bool hasEquivalentVariable(const VariablePtr &equivalentVariable) const;
 
+    /**
+     * @brief Test whether the argument variable is equivalent to this variable.
+     *
+     * Tests to see if the argument variable is equivalent to this variable.
+     * Returns @c true if the argument variable is equivalent to this variable
+     * and @c false otherwise.
+     *
+     * @param equivalentVariable The variable to check whether it is equivalent this variable.
+     *
+     * @return @c true if the @p equivalentVariable is equivalent to this variable
+     * and @c false otherwise.
+     */
+    bool isEquivalentVariable(const VariablePtr &equivalentVariable) const;
```

Comment:
This is exactly the same as `hasEquivalentVariable`?? We don't need both just a properly working one.

Reply:
Oops, my bad, I indeed completely overlooked your ["old" comment](https://github.com/cellml/libcellml/issues/321#issuecomment-491995223).

Ok, I am going to remove the contents of the "old" `hasEquivalentVariable()` and replace it with my new one.

Reply:
Sounds good.